LocalSettings.php

```diff
@@ -4430,6 +4440,22 @@
 				'editmyoptions' => true,
 			),
 		),
+		'wikicanadawiki' => array(
```

Comment:
@Reception123 I know that RevokePermissions works, because you aren't supposed to set anything to "true" in LocalWiki (I believe)